react/features/settings/components/web/ProfileTab.js

```diff
@@ -177,6 +179,8 @@ class ProfileTab extends AbstractDialogTab<Props> {
             t
         } = this.props;
 
+        const { DISABLE_SUPERHERO = false } = interfaceConfig;
```

Comment:
not sure that we need default value.

Reply:
destructuring will return the `undefined`
```js
const { s } = {};
// undefined
s
// undefined
```

Reply:
so the next condition will work as expected
https://github.com/aeternity/jitsi-meet/blob/f86fb4a3f5f2c90dd68a8b4e75c077743c972a7b/react/features/settings/components/web/ProfileTab.js#L214-L222

and we don't have any interfaceConfig schema or type check except above so it can be any shape and include any types
https://github.com/aeternity/jitsi-meet/blob/f86fb4a3f5f2c90dd68a8b4e75c077743c972a7b/react/features/settings/components/web/ProfileTab.js#L19